client/gutenberg/extensions/publicize/panel.jsx

```diff
@@ -54,12 +54,12 @@ class PublicizePanel extends Component {
 				}
 			>
 				<div>{ __( 'Connect and select social media services to share this post.' ) }</div>
-				{ ( connections.length > 0 ) && <PublicizeForm staticConnections={ connections } refreshCallback={ getConnectionsStart } /> }
-				{ ( 0 === connections.length ) && <PublicizeNoConnections refreshCallback={ getConnectionsStart } /> }
+				{ ( connections && connections.length > 0 ) && <PublicizeForm staticConnections={ connections } refreshCallback={ getConnectionsStart } /> }
```

Comment:
`_.get( connections, [ 'length' ], 0 )`?

Reply:
Or maybe drop the extra guard, if you initialize `connections` as empty array [rather than `null`](https://github.com/Automattic/wp-calypso/pull/28050/files#diff-97655b35140fa59d245d71fa321f4413R12).

Reply:
I'm inclined to leave as-is, as `_.get()` will add just another dependency, and empty array will show all the services for a little while the connections are loading.